iron-fit-behavior.html

```diff
@@ -171,17 +278,14 @@
      * the memoized data.
      */
     resetFit: function() {
-      if (!this._fitInfo || !this._fitInfo.sizedBy.width) {
-        this.sizingTarget.style.maxWidth = '';
+      var info = this._fitInfo || {};
+      for (var property in info.sizerInlineStyle) {
```

Comment:
You're right, `in` only throws when used in an `if` for the condition that I described.

Carry on!